engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -489,10 +491,13 @@ private void updateCamera(CharacterMovementComponent charMovementComp, Vector3f
             }
         }
 
-        if (charMovementComp.mode == MovementMode.GHOSTING) {
-            playerCamera.extendFov(24);
-        } else {
-            playerCamera.resetFov();
+        if(charMovementComp.mode != lastMode){
+            if (charMovementComp.mode == MovementMode.GHOSTING) {
```

Comment:
IIRC it is literally the `ghost` movement mode you can enter via the game console. It changes the camera a bit.